Code/GraphMol/DistGeomHelpers/BoundsMatrixBuilder.cpp

```diff
@@ -815,12 +815,21 @@ bool _checkAmideEster14(const Bond *bnd1, const Bond *bnd3, const Atom *atm1,
   unsigned int a2Num = atm2->getAtomicNum();
   unsigned int a3Num = atm3->getAtomicNum();
   unsigned int a4Num = atm4->getAtomicNum();
-
-  if (a1Num != 1 && a3Num == 6 && bnd3->getBondType() == Bond::DOUBLE &&
+  // std::cerr << " -> " << atm1->getIdx() << "-" << atm2->getIdx() << "-"
+  //           << atm3->getIdx() << "-" << atm4->getIdx()
+  //           << " bonds: " << bnd1->getIdx() << "," << bnd3->getIdx()
+  //           << std::endl;
+  // std::cerr << "   " << a1Num << " " << a3Num << " " << bnd3->getBondType()
+  //           << " " << a4Num << " " << bnd1->getBondType() << " " << a2Num <<
+  //           " "
+  //           << atm2->getTotalNumHs(true) << std::endl;
+  if (a3Num == 6 && bnd3->getBondType() == Bond::DOUBLE &&
       (a4Num == 8 || a4Num == 7) && bnd1->getBondType() == Bond::SINGLE &&
-      (a2Num == 8 || (a2Num == 7 && atm2->getTotalNumHs() == 1))) {
+      (a2Num == 8 || (a2Num == 7 && atm2->getTotalNumHs(true) == 1))) {
```

Comment:
Is this fix related to trans amids?

Reply:
Yes, the fact that the wrong version of getTotalNumHs() was being called meant that some amides were not being correctly identified